### PR TITLE
Add doc for Lab/Badge

### PR DESCRIPTION
**Pull request by LeoLYW12138**
Add doc page for Lab/Badge component as requested by https://github.com/jpmorganchase/uitk/issues/161

### CODE REVIEW

File: packages/lab/stories/badge.doc.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  title="Documentation/Lab/Badge"
+  component={Badge}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Badge
+
+The Badge component provides a small status descriptor for UI elements. It is a small circle containing a number or a few characters that typically appears in one of the corners of the UI element.
```
Comment (libertymayc) [inline packages/lab/stories/badge.doc.stories.mdx:27]: Change wording to "A Badge is numeric annotation that indicates the number of outstanding items that need to be addressed. It appears on the top-right of an element, usually an icon. The badge can accommodate values of up to four digits in length."

File: packages/lab/stories/Badge.stories.newapp-badge.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.uitk-success.uitk-newapp {
```
Comment (libertymayc) [inline packages/lab/stories/Badge.stories.newapp-badge.css:1]: we should provide these as a className to the badge ("uitkBadge-success" on the component) and then have .uitk-newapp .uitkBadge-success {  
  --uitkBadge-content-icon-color: var(--uitk-status-success-foreground);
  --uitkBadge-content-text-color: var(--uitk-color-white);
}
and same for error! :)

File: packages/lab/stories/badge.stories.tsx [hunk 3]
```diff
@@ -62,3 +65,14 @@ WordsBadge.args = {
   badgeContent: 1,
   children: "Lorem Ipsum",
 };
+
+export const CustomStyling: ComponentStory<typeof Badge> = () => (
+  <div style={{ display: 'flex', gap: 5 }}>
+    <ToolkitProvider theme={["success", 'newapp']}>
```
Comment (libertymayc) [inline packages/lab/stories/badge.stories.tsx:71]: we need to keep "light" or "dark" in the theme to get the rest of the styles (in storybook, we luckily have them coming from higher up)
then, let's add 2 Badge's to both light and dark, one with className 'success' and one with 'error'! :)

**Review by libertymayc — CHANGES_REQUESTED** [review]
Changes requested in comments

**Comment by LeoLYW12138** [comment]
Thank you for your review! Sorry for any silly mistakes as I'm a first-time contributor.

**Comment by LeoLYW12138** [comment]
I think I did something wrong, suddenly there are merge conflicts on my branch. Is this normal?

**Comment by libertymayc** [comment]
> I think I did something wrong, suddenly there are merge conflicts on my branch. Is this normal?

You just need to rebase on the main branch, as there have been some changes today :)

**Comment by origami-z** [comment]
@LeoLYW12138 Thanks for the contribution. In case the CLA process is not clear for you (the label I added)

We requrie all contributors to JPMC projects to complete our Contribution Licence Agreement (CLA). You can sign as an individual or as representing your company. You only need to do this once and it covers all current and future projects managed under our GitHub organization.

The contribution agreement can be found here [JPMC CLA](https://github.com/jpmorganchase/.github/blob/main/Contribution%20License%20Agreement%20(CLA)%20JPMC%202022.pdf), follow the instructions to complete and return to [opensource_cla@jpmorgan.com](mailto:opensource_cla@jpmorgan.com)

**Comment by LeoLYW12138** [comment]
Sorry for the inconvenience but I have signed the CLA and returned to opensource_cla@jpmorgan.com through email on Friday noon GMT+8. However, there is no response or further notice.

**Comment by origami-z** [comment]
I will chase up and get this merged ASAP. Thank you for the patience. 👍

**Comment by origami-z** [comment]
CLA is handled by a different team. I have chased up again.

**Review by origami-z — APPROVED** [review]

**Review by origami-z — APPROVED** [review]

**Pull request merged**